indico/htdocs/sass/modules/abstracts/_abstract_list.scss

```diff
@@ -24,3 +24,18 @@
         color: #777;
     }
 }
+
+.back-button {
+    position: relative;
+
+    a {
+        position: absolute;
+        top: 0.15em;
+        left: -1.2em;
+        font-size: 0.9em;
+        color: $dark-gray;
```

Comment:
`@import 'base';` at the beginning of the file (like in most other scss files in the same folder), otherwise this is undefined.